### PR TITLE
ENG-1408: When creating/editing a Page I should be see a thumbnail to select the Page Template

### PR DESCRIPTION
**Pull request by gReis89**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **greis89**

If they are not already assigned, you can assign the PR to them by writing `/assign @greis89` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/locales/it.js
Comment (gidesan) [inline src/locales/it.js:469]:
```suggestion
    'pages.pageForm.findTemplate': 'Seleziona un template',
```

File: src/locales/it.js
Comment (gidesan) [inline src/locales/it.js:468]:
```suggestion
    'pages.pageForm.pageTemplateHelp': 'Seleziona un template di pagina da utilizzare',
```

File: src/ui/pages/common/const.js
Comment (gidesan) [inline src/ui/pages/common/const.js:38]: please extract process.env.PUBLIC_URL to a variable

**Review by gidesan — CHANGES_REQUESTED** [review]
almost good, only very little changes requested :)

**Review by gidesan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/896#pullrequestreview-521848713" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
